workloads/templates/common-templates.md

```diff
@@ -0,0 +1,55 @@
+# Virtual machine templates
+
+## What is a virtual machine template?
+
+The KubeVirt projects provides a set of [templates](https://docs.okd.io/latest/dev_guide/templates.html) to create VMs to handle common usage scenarios.
+These templates provide a combination of some key factors that could be further customized and processed to have a Virtual Machine object.
+The key factors which define a template are
+
+- Workload
+  Most Virtual Machine should be *generic* to have maximum flexibility; the *highperformance* workload trades some of this flexibility to
+  provide better performances.
+- Guest Operating System (OS)
+  This allow to ensure that the emulated hardware is compatible with the guest OS. Furthermore, it allows to maximize the stability
+  of the VM, and allows performance optimizations.
+- Size (flavor) 
+  Defines the amount of resources (CPU, memory) to allocate to the VM.
+
+More documentation is available in the [common templates subproject](https://github.com/kubevirt/common-templates)
+
+## Accessing the virtual machine templates
+
+If you installed KubeVirt using a supported method you should find the common templates preinstalled in the cluster.
+Should you want to upgrade the templates, or installed them from scratch, you can use one of the [supported release](https://github.com/kubevirt/common-templates/releases)
+
+To install the templates:
+```bash
+$ export VERSION="v0.3.1"
+$ oc create -f https://github.com/kubevirt/common-templates/releases/download/$VERSION/common-templates-$VERSION.yaml
+```
+
+## Editable fields
+
+You can edit the fields of the templates which define the amount of resources which the VMs will receive.
```

Comment:
Each template can list a different list of fields that are to be considered editable. It serves as a hint to the UI and to whatever upgrade logic we will have. So either frame the list below as an example (and make it more obvious it is just an example) or remove it please. Also you should actually say where the list is defined (the annotation name).

Reply:
This is still unresolved.

Reply:
yes, I missed this comment. On it now.